tests/retryWrites-toggleServerSideJS.yml

```diff
@@ -0,0 +1,64 @@
+initialConfiguration:
+  clusterConfiguration:
+    clusterType: REPLICASET
+    providerSettings:
+      providerName: AWS
+      regionName: US_WEST_1
+      instanceSizeName: M10
+  processArgs: {}
+
+operations:
+  -
+    clusterConfiguration: {}
+    processArgs:
+      javascriptEnabled: true
+
+driverWorkload:
+  description: "Insert"
+
+  schemaVersion: "1.0"
+
+  createEntities:
+    - client:
+        id: &client0 client0
+        uriOptions:
+          retryWrites: true
```

Comment:
How are the uriOptions actually going to be honored? Astrolabe doesn't seem to use this field and we prohibit the workload executor from using it.

Reply:
This field is read by the unified test runner when it is instantiating clients.

Reply:
Ok, in that case we need to update the language in `docs/source/spec-workload-executor.rst`. Currently, that document states:

```
Behavioral Description
----------------------

After accepting the inputs, the workload executor:

#. MUST use the input connection string to `instantiate the
   unified test runner <https://github.com/mongodb/specifications/blob/master/source/unified-test-format/unified-test-format.rst#id92>`_
   of the driver being tested. Note that the workload executor:

   * MUST NOT override any of the URI options specified in the incoming connection string.
   * MUST NOT augment the incoming connection string with any additional URI options.
```

If we want the uriOptions to be honored, we need to update this language?

Reply:
I don't think the language you quoted conflicts. It's saying that the workload executor must pass the connection string it receives without modification to the unified test runner. AFAIK, the workload executor isn't connecting to MongoDB directly, so the only client objects using this connection string will be those created _within_ the unified test runner. IMO, this has no bearing on the unified test runner spec's instructions that a client entity's `uriOptions` should be respected.

Perhaps a clarification of this interaction would help avoid any misunderstanding.